src/server/pachyderm_test.go

```diff
@@ -92,6 +93,70 @@ func TestPipelineWithParallelism(t *testing.T) {
 	}
 }
 
+func TestPipelineWithLargeFiles(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	defer require.NoError(t, c.DeleteAll())
+
+	dataRepo := uniqueString("TestPipelineInputDataModification_data")
+	require.NoError(t, c.CreateRepo(dataRepo))
+
+	r := rand.New(rand.NewSource(99))
+	numFiles := 10
+	var fileContents []string
+
+	commit1, err := c.StartCommit(dataRepo, "master")
+	require.NoError(t, err)
+	for i := 0; i < numFiles; i++ {
+		fileContent := workload.RandString(r, int(pfs.ChunkSize)+i*MB)
```

Comment:
This is a fine way to do this but `workload` actually has a `NewReader` method which will give this to you as a reader rather than a string you have to convert into a reader. Advantage is slightly less code and your test will use less memory which may make it run slightly faster on CI.

Reply:
I need to store the strings so I can compare them later.  Tho I suppose I could also create a new RNG and generate the same readers again, but that sounds more complicated.